token-lending/README.md

```diff
@@ -27,23 +27,23 @@ This is optional! You can skip these steps and use the [Token Lending CLI](./cli
 
 1. [Install the Solana CLI](https://docs.solana.com/cli/install-solana-cli-tools)
 
-1. Install the Token and Token Lending CLIs:
+2. Install the Token and Token Lending CLIs:
```

Comment:
prefer not to make this change. markdown does some magic and autoincrements numbered lists where all items are numbered `1.`.  this allows for reordering the list with smaller diffs

Reply:
My bad - I wasn't aware that markdown had this feature. Fixed!